### DISCUSSION TITLE
How can I completely disable the chrome:// suggestions in the search bar?

### DISCUSSION THREAD

**Discussion by ungubajue90k** [ungoogled-software/ungoogled-chromium-android#50] [Q&A]
Hello,

i use this latest ungoogled-chromium android version
https://github.com/ungoogled-software/ungoogled-chromium-android/releases/tag/87.0.4280.141-2-extensions

if i type `a`, `c` or `ch` in the search bar immediately these suggestions appear: 

- chrome://chrome-urls/
- chrome://version/

![IMG_20210126_160418](https://user-images.githubusercontent.com/77856086/105863201-eee97880-5ff0-11eb-9055-2423f8b77e9b.jpg)
![IMG_20210126_160758](https://user-images.githubusercontent.com/77856086/105863224-f446c300-5ff0-11eb-954f-5a8c8a41052a.jpg)

How can I disable this completely?

**Comment by wchen342** [comment]
There is no such filter I think.

**Reply by ungubajue90k** [reply]
> There is no such filter I think.

How can it be fixed? I do not like these chrome:// suggestions. Do not want such suggestions unless I decide that.

Can you add an option to disable it?

**Reply by wchen342** [reply]
This should be implemented upstream I think. Can you open a feature request in the [main repo](https://github.com/Eloston/ungoogled-chromium)?

**Reply by ungubajue90k** [reply]
> This should be implemented upstream I think. Can you open a feature request in the [main repo](https://github.com/Eloston/ungoogled-chromium)?

Done.
https://github.com/Eloston/ungoogled-chromium/issues/1356

**Reply by wchen342** [reply]
The patch has been merged [upsteam](https://github.com/Eloston/ungoogled-chromium/pull/1378) and will be available in next regular (not extension-support version only) release.
